<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simplifyLineDialog</class>
 <widget class="QDialog" name="simplifyLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simplify Line</string>
  </property>
  <widget class="Q3ButtonGroup" name="polygonToPolyLineGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>570</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="Q3Table" name="inputOutputTable" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>200</height>
     </rect>
    </property>
    <property name="lineWidth" stdset="0">
     <number>2</number>
    </property>
    <property name="midLineWidth" stdset="0">
     <number>0</number>
    </property>
    <property name="margin" stdset="0">
     <number>0</number>
    </property>
    <property name="dragAutoScroll" stdset="0">
     <bool>false</bool>
    </property>
    <property name="numRows" stdset="0">
     <number>0</number>
    </property>
    <property name="numCols" stdset="0">
     <number>3</number>
    </property>
    <property name="showGrid" stdset="0">
     <bool>true</bool>
    </property>
    <property name="rowMovingEnabled" stdset="0">
     <bool>false</bool>
    </property>
    <property name="columnMovingEnabled" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFileButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>removeFile</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../PolygonToPolyLine/polygontopolylinedialog.qrc">
      <normaloff>:/icons/Minus32.png</normaloff>:/icons/Minus32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>remove All Files</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../PolygonToPolyLine/polygontopolylinedialog.qrc">
      <normaloff>:/icons/Delete32.png</normaloff>:/icons/Delete32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="editToleranceButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Edit Tolerance</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../PolygonToPolyLine/polygontopolylinedialog.qrc">
      <normaloff>:/icons/Properties32.png</normaloff>:/icons/Properties32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3ButtonGroup" name="buttonGroup8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>570</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="InputPolyline">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>110</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Polyline</string>
    </property>
   </widget>
   <widget class="QLabel" name="tolerance">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>110</width>
      <height>48</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Simplification Tolerance</string>
    </property>
    <property name="text">
     <string>Tolerance (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputPolylineEdit">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>20</y>
      <width>341</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="toleranceEdit">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>70</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addFileButton">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>68</y>
      <width>81</width>
      <height>37</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add (Polyline, Tolerance) to Table</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../PolygonToPolyLine/polygontopolylinedialog.qrc">
      <normaloff>:/icons/Plus32.png</normaloff>:/icons/Plus32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="inputPolylineButton">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>20</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="MessageLog" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>417</y>
     <width>570</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>379</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>379</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>379</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends></extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../PolygonToPolyLine/polygontopolylinedialog.qrc"/>
 </resources>
 <connections/>
</ui>
